src/main/webapp/app/lecture/wizard-mode/lecture-wizard-period.component.html

```diff
@@ -6,19 +6,23 @@ <h1><span jhiTranslate="artemisApp.lecture.wizardMode.steps.periodStepTitle">Per
             <jhi-date-time-picker
                 labelName="{{ 'artemisApp.lecture.startDate' | artemisTranslate }}"
                 [(ngModel)]="lecture.startDate"
-                (valueChange)="validateDatesFunction()"
+                (valueChange)="validateDatesFunction(); update()"
```

Comment:
I tried rewriting update() so that validateDatesFunction would be called from it but it did not update the isEndDateBeforeStartDate for some reason. In update emitting is only done in case isEndDateBeforeStartDate is true. I don't really like the "validateDatesFunction(); update()" line that much, I would prefer if there only was one function call, but as I mentioned it did not work when I tried it.

Reply:
Hmm, alright. Then let's leave it for now with renaming `update` to something more fitting.
`emitInvalidDateConfiguration` maybe? Or anything that describes the behaviour more propertly.

Reply:
done :+1: